### ISSUE TITLE
Train speed is very slow!

### ISSUE THREAD

**Issue by dreamer121121** [JizhiziLi/GFM#27]
I try to add ur focus branch to my own branch but i found that the train speed will be very slow, i found that in loss.backward() and optimizer.step() will cost much time about 50S for [32, 3, 512, 512] input.

**Comment by JizhiziLi** [comment]
Hi there, I don't know how you have merged the focus branch with your own branch exactly but the training speed in our experiments should not be that slow. It takes about two days for 500 epochs when training on AM-2k. Please note that the CM module is different when in training and in testing, which could be part of the reasons for the slow training speed.

BTW, the complete training code will be released very soon recently, you can try it out to see if it can improve the training speed on your side. Thanks.
